src/routing.js

```diff
@@ -119,15 +119,16 @@ export function getAppPath({
 }
 
 // Preferences
-const GLOBAL_PREFERENCES_QUERY_PARAM = '?preferences=/'
+const GLOBAL_PREFERENCES_QUERY_PARAM = '?preferences='
 const GLOBAL_PREFERENCES_SHARE_LINK_QUERY_VAR = '&labels='
 
 function parsePreferences(search = '') {
-  const [, raw = ''] = search.split(GLOBAL_PREFERENCES_QUERY_PARAM)
   const params = new Map()
```

Comment:
Hmm yes, this might go into the routing refactor that @bpierre is working on, but instead of using this Map, we can just pass down the URLSearchParams.